java/clients/src/main/java/sleeper/clients/QueryWebSocketClient.java

```diff
@@ -66,30 +66,32 @@
 
 public class QueryWebSocketClient extends QueryCommandLineClient {
     private final String apiUrl;
-    private final Client client;
+    private final InstanceProperties instanceProperties;
+    private final TablePropertiesProvider tablePropertiesProvider;
+    private final ConsoleOutput out;
```

Comment:
It doesn't look like these 3 fields are used?

Reply:
Thanks, the ConsoleOutput is still used in reporting the query duration, but i've removed the other 2 fields.